.github/workflows/master.yml

```diff
@@ -97,6 +97,14 @@ jobs:
       build_name: package_release
       checkout_depth: 0
       data: ${{ needs.RunConfig.outputs.data }}
+  BuilderDebReleaseCoverage:
```

Comment:
You need to add BuilderDebReleaseCoverage to the BuilderReport job below. Here and in the pull_request.yml